<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>지균이와 감자들</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../사진 및 그림/icon.png</normaloff>../사진 및 그림/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : #FFFFFF;</string>
  </property>
  <widget class="QLabel" name="program_name">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Program Name</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../사진 및 그림/톱니바퀴.png</normaloff>../사진 및 그림/톱니바퀴.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="loopAni">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="text">
    <string>Loop Animaition</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="answer_edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="command_edit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="mic_btn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>170</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../사진 및 그림/마이크.png</normaloff>../사진 및 그림/마이크.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="hidden_mic_btn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: none;  /* 테두리 제거 */
    background-color: transparent;  /* 배경을 투명하게 설정 */
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
